ssh/DOCS.md

```diff
@@ -1,55 +1,63 @@
-# Home Assistant Add-on: SSH server
+# Home Assistant Add-on: SSH Server
 
 ## Installation
 
 Follow these steps to get the add-on installed on your system:
 
 1. This addon is only visible to "Advanced Mode" users. To enable advanced mode, go to **Profile** -> and turn on **Advanced Mode**.
 2. Navigate in your Home Assistant frontend to **Supervisor** -> **Add-on Store**.
-3. Find the "SSH server" add-on and click it.
+3. Find the "Terminal & SSH" add-on and click it.
 4. Click on the "INSTALL" button.
 
 ## How to use
 
-You need enable the port for external access. You can just enter 22 as value or any other
-value like you want. This recommend to add login credentials.
+This add-on enables two main features to your Home Assistant installation:
+- a Terminal that you can use from your browser, and
+- enable connecting to your system using an SSH client.
 
-To use this add-on, you must have a private/public key to log in.
-To generate them, follow the [instructions for Windows][keygen-windows]
-and [these for other platforms][keygen]. It is possible to set a password for
-login since version 2.0 but for high security use private/public keys.
+Regardless of how you connect (using the Terminal, or a using an SSH client), you end up in this add-on's container. The Home Assistant configuration 
+directory is mounted on the path `/config`.
```

Comment:
Maybe avoid the term "mounted" here, as it might be confusing (as one can not control the mounts or might not even know what a mount is).

```suggestion
directory is located on the path `/config`.
```

Reply:
Absolutely agree!